wfe/web-front-end.go

```diff
@@ -258,10 +272,20 @@ func (wfe *WebFrontEndImpl) verifyPOST(request *http.Request, regCheck bool) ([]
 	return []byte(payload), key, reg, nil
 }
 
+func (wfe WebFrontEndImpl) verifyRateLimits(key crypto.PublicKey) (error, time.Duration) {
```

Comment:
We should call this from verifyPOST rather than from each individual Handle method. And in verifyPOST, for most cases we have a registration id, which would be nicer to use than the hash of the account key since we don't have to hash it, and because it is stable even if someone is changing out their account key.

For the situations where verifyPOST does not require a valid registration (new-reg and revoke), we could instead bucket by IP address (doesn't need to be in this change, though).